server/services/operations_service.go

```diff
@@ -164,7 +164,7 @@ func (s operationsServerImpl) ListOperations(ctx context.Context, in *lropb.List
 
 	return &lropb.ListOperationsResponse{
 		Operations:    operations,
-		NextPageToken: "txenNext",
+		NextPageToken: "",
```

Comment:
Just remove this line entirely, it will default to an empty string.

Reply:
Sure, done.